packages/artifact/src/internal/artifact-client.ts

```diff
@@ -173,6 +173,7 @@ export class DefaultArtifactClient implements ArtifactClient {
       await createDirectoriesForArtifact(
         downloadSpecification.directoryStructure
       )
+      core.info('Directory structure has been setup for the artifact')
```

Comment:
Is it worth including the name of the directory that was created?

Reply:
Not worth it. There is actually more than 1 directory that gets created and this all happens before any downloading actually starts.